robottelo.properties.sample

```diff
@@ -187,8 +187,8 @@ ssh_key=
 
 # For tests that uses the images for content-host testcases.
 # [distro]
-# image_el6=rhel68
-# image_el7=rhel73
+# image_el6=rhel610
```

Comment:
Not requires to update this one as RHEL6.10 is latests one ..

Reply:
@vijay8451 Can you please explain to me, why we do not need to change this ? I am probably missing the idea which version should be in distro section.

Reply:
ahh .. you can ignore ... I missed the version change rhel610 :)